.github/workflows/platformDeploy.yml

```diff
@@ -0,0 +1,270 @@
+name: Build and deploy android, desktop, iOS, and web clients
+
+# This workflow is run when any tag is published
+on:
+  push:
+    tags:
+      - '*'
+  release:
+    types: [created]
+
+env:
+  SHOULD_DEPLOY_PRODUCTION: ${{ github.event_name == 'release' }}
+  DEVELOPER_DIR: /Applications/Xcode_12.3.app/Contents/Developer
+
+jobs:
+  android:
+    name: Build and deploy Android
+    if: github.actor == 'OSBotify'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+
+      - name: Install bundler
+        run: gem install bundler
+
+      - name: Install gems
+        run: bundle install
```

Comment:
NAB but not sure why we don't have this wrapped in a retry action

Reply:
I don't think it's ever failed, but we can add a retry

Reply:
Okay, no worries for now